src/bin/mir2wasm.rs

```diff
@@ -1,5 +1,4 @@
-#![feature(rustc_private, custom_attribute, link_args)]
-#![allow(unused_attributes)]
```

Comment:
I'm glad you were able to get rid of `allow(unused_attributes)`. It seems good for code hygiene.